system-int-test/src/cmr/system_int_test/utils/generic_util.clj

```diff
@@ -37,7 +37,7 @@
                             (slurp)
                             (json/parse-string true)))
 
-(def variable-draft (-> "schemas/variable-draft/v1.0.0/metadata.json"
+(def variable-draft (-> "schemas/variable-draft/v1.0.0/metadata_with_private_data.json"
```

Comment:
I can't read Clojure, but do you need a test to verify that the _private doesn't show up in the resulting metadata?

Reply:
No. I mentioned it at the very beginning that tests are not needed. The existing tests covered it.  If the _private is not removed, the schema validation would fail when publishing the variable-draft because the element doesn't exist in the schema.